mne/io/brainvision/brainvision.py

```diff
@@ -488,20 +490,31 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     misc = list(misc_chs.keys()) if misc == 'auto' else misc
 
     # create montage
-    if montage is True:
-        from ...transforms import _sphere_to_cartesian
+    if cfg.has_section('Coordinates') and montage is None:
+        from ...transforms import _sph_to_cart
         from ...channels.montage import Montage
         montage_pos = list()
         montage_names = list()
+        to_misc = list()
         for ch in cfg.items('Coordinates'):
-            montage_names.append(ch_dict[ch[0]])
+            ch_name = ch_dict[ch[0]]
+            montage_names.append(ch_name)
             radius, theta, phi = map(float, ch[1].split(','))
             # 1: radius, 2: theta, 3: phi
-            pos = _sphere_to_cartesian(r=radius, theta=theta, phi=phi)
+            pol = np.deg2rad(theta)
+            az = np.deg2rad(phi)
+            pos = _sph_to_cart(np.array([[radius * 85., az, pol]]))[0]
```

Comment:
why is the radius multiplied by `0.85` here?
And what makes us sure that theta and phi are given in radians and not degrees?

Reply:
multiplier is `85.0`, not `0.85`.  But that doesn't answer your question.

Reply:
for the second question: the code assumes `theta` and `phi` *are* in degrees, and they get transformed to radians on the lines above (when they're converted to `pol` and `az`).

Reply:
most likely a hack that works. See: https://github.com/mne-tools/mne-python/pull/4493#discussion_r134666572

Reply:
hehe, apparently I am not the first one to misread the `85` as a `0.85`, see Alex's comment: https://github.com/mne-tools/mne-python/pull/4493#discussion_r134572219 and yes, thanks for the correction @drammock --> but do you know **why**?

Thanks for linking to that comment Mainak ... although this leads to a further comment https://github.com/mne-tools/mne-python/pull/4493#discussion_r134666572 saying that this Hack was taken from https://github.com/mne-tools/mne-python/blob/033f8c13fb03772c26142a1782c18a2a65dc9e8e/mne/channels/montage.py#L323-L331

So my question remains unanswered. I'll continue the discussion in https://github.com/mne-tools/mne-python/pull/4109